### PR TITLE
Add Mixi.Media AD network widgets extension 

### PR DESCRIPTION
**Pull request by isublimity**
# Overview:
✨ New feature 

Provides support for amp-embed type=miximedia
It will be widget with content recommendations for User.
Project site https://mixi.media/

Mixi.Media is a service for personalizing content and increasing user metrics ad network.

# Example horizontal block

Example horizontal block Mixi.Media

# Usage
```xml
<amp-embed height="284"
        type="miximedia"
        data-blockid="91908">
</amp-embed>
```

### CODE REVIEW

File: ads/miximedia.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    startTime: window.context.startTime,
+
+  });
+  global._miximedia.AMPCallbacks = {
+    renderStart: global.context.renderStart,
+    noContentAvailable: global.context.noContentAvailable,
+  };
+  // load the miximedia  AMP JS file script asynchronously
+  const rand = Math.round(Math.random() * 100000000);
+  loadScript(global, 'https://static.mixi.media/static/amppages/ampclient/mixi.js?rand=' + rand, () => {},
```
Comment (zhouyx) [inline ads/miximedia.js]: nit: 80 chars limit

File: ads/_config.js [hunk 2]
```diff
@@ -609,6 +609,9 @@ export const adConfig = {
       'https://deb.send.microad.jp',
     ],
   },
+  miximedia: {
```
Comment (zhouyx) [inline ads/_config.js]: nit: empty line above

**Comment by isublimity** [comment]
cla:yes

**Comment by isublimity** [comment]
@zhouyx I`m fix `empty line` and `80 chars limit`
Thx!

**Comment by zhouyx** [comment]
Thank you @isublimity. PR merged.

**Pull request merged**